app/views/posts/show.html.erb

```diff
@@ -1,15 +1,21 @@
 <h3 style="color: #FF4433;">User Post</h3>
 <p> Your Post data: <table>
-<tr><th>Name User ID </th><th>Title </th><th>Content </th> <th>Commnets</th><th>Likes</th></tr> 
- <tr>
- <td><%= @post[:user_id] if @post %> </td>
- <td><%= @post[:title]  if @post %> </td>
- <td><%= @post[:text]  if @post %> </td>
- <td><%= @post[:comments_counter]  if @post %> </td> 
- <td><%= @post[:likes_counter] if @post %> </td> 
- </tr>
-</table>
- </p>
-<p> Current Post: <%= @post[:title]  if @post %> </p>
-<p>  Content : <%= @post[:text] if @post %>  </p>
+
+  <div style="background: rgba(255, 213, 128, 0.1); width: 80%;" padding: 10px; margin: 5px; border-radius: 5px;%>
+    <h3>Post &#35;<%=@post.id %> by <%= @user.name %>  <span style="color: #FF4433; font-size: 12px;">Comments: <%= @post.comments_counter %> </span> <span style="color: #FF4433; font-size: 12px;">Likes: <%= @post.likes_counter %> </span> </h3>
+    <p style= "color: #900C3F; padding: 5px; font-size: 24px;"> <%= @post.text %></p>
+  </div>
+  <div> 
+  <% @post_comments.each do |comment| %>
+    <% @post_users.each do |user| %>
+     <% if comment.post_id == @post.id && user.id == comment.user_id %>
+     <p style= "color: #900C3F; padding: 5px;">
+     <%= user.name %>  
+     <%= comment.text %> 
+     </p>
+     <% end %>
+      <% end %>
+    <% end %>
+  </div>
+
```

Comment:
- [ ] Lastly, kindly make sure that post show page looks like the wireframe below:

![Screenshot from 2022-08-26 20-01-02](https://user-images.githubusercontent.com/63087491/186973897-aa560222-1058-4dba-8208-c890f3bd43ad.png)

Reply:
@FrederickMih  Added suggested changes 👍🏽 
![image](https://user-images.githubusercontent.com/94127418/187027083-e9697d43-c3d6-47d5-ac2d-c62d632e650c.png)